### PR TITLE
Smack the cancer-stick right outta someone's mouth!

### PR DESCRIPTION
**Pull request by Tacolizard**
:cl: Tacolizard Forever
add: You can now smack the cigarette out of someone's mouth by targeting their mouth and disarming them.
/:cl:
 idea from: https://tgstation13.org/phpBB/viewtopic.php?f=9&t=30&start=3100#p366695

### CODE REVIEW

**Comment by AlexTheSysop42** [comment]
Cigs cause cancer when?

**Comment by Tacolizard** [comment]
@AlexTheSysop42 hmmm.

I'm not sure about cancer but maybe I could make them damage your lungs.

**Comment by Qustinnus** [comment]
this also stops them from getting good smoking moodies

**Comment by Wilchenx** [comment]
Make it so you slap someone when they don't have a cigarette and we're golden.

**Comment by Tacolizard** [comment]
@Wilchenx i think that may have been a feature in the past, and I don't particularly care to add it.

**Comment by Wilchenx** [comment]
yeah but current slapping is shitcode and we can integrate moodies into it

**Comment by Cyberboss** [comment]
So long as doing it makes you valid

**Comment by optimumtact** [comment]
Do not add slapping

**Comment by Wilchenx** [comment]
Don't listen to optimumtact, she didn't take her phoebe pills yet

**Comment by Tacolizard** [comment]
@optimumtact i wasn't planning on it

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1253,6 +1253,16 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 			target.forcesay(GLOB.hit_appends)
 
 /datum/species/proc/disarm(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
+	if(user.zone_selected == "mouth" && istype(target.wear_mask, /obj/item/clothing/mask/cigarette))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species.dm]: This is snowflake af. It's something specific to cigarettes, so it should be on the cigarette path

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species.dm (hunk 1 above)
Comment (Tacolizard) [reply code/modules/mob/living/carbon/human/species.dm]: the only way i can think of doing that is adding code to the cigarette's process() to check if its user has been disarmed by someone targeting the mouth. is that a good solution?

**Comment by AnturK** [comment]
You realize this is already a thing ? https://github.com/tgstation/tgstation/pull/33594

**Comment by Tacolizard** [comment]
@AnturK uhhhhhh

i could've sworn that you can't already do this. are you sure it's not bugged?

**Comment by AnturK** [comment]
Someone probably broke it since i added it. I'll take a look in a moment

**Comment by Cyberboss** [comment]
Duplicate of #33594

**Pull request closed without merging**